### PR TITLE
Improve message formatting in some strings

### PR DESCRIPTION
**Pull request by GNosii**
#### Description: 

This PR is meant to improve message formatting in [NationCommand.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/command/NationCommand.java), [TownyAdminCommand.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java), [TownCommand.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/command/TownCommand.java) and [TownyFormatter.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/TownyFormatter.java) by changing how some strings are formatted, making them more user-friendly and nice looking.
____

#### Changes made:
- Use `TownyMessaging.sendMsg` instead of `TownyMessaging.sendMessage`.
- Make nation and town messages use `TownyMessaging.sendPrefixedNationMessage` and `TownyMessaging.sendPrefixedTownMessage` whenever it might be useful.
- Make invite/ally request messages be nation/town messages instead of them being only shown to the user that revoked them.
- Make money-related messages use `TownyEconomyHandler.getFormattedBalance` to make them more user-friendly. (Example: "Player deposited 50000 into the town bank." is now displayed as "Player deposited $50,000 into the town bank.")
- Make [TownyAdminCommand.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java) use prefixed **[Towny]** messages when possible.
- Make [TownyFormatter.java](https://github.com/TownyAdvanced/Towny/blob/master/src/com/palmergames/bukkit/towny/TownyFormatter.java) return a different status screen for NPC residents.
____
#### New Nodes/Commands/ConfigOptions: 

____
#### Relevant Towny Issue ticket:

____
- [x] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

**Comment by docwhomc** [comment]
Your commit messages could use some work. "Si" and "added a new line" are neither informative nor accurate.

**Comment by GNosii** [comment]
> 
> 
> Your commit messages could use some work. "Si" and "added a new line" are neither informative nor accurate.

I do know, I had an small issue with git commiting so yeah, I'll try to improve commit messages

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 1]
```diff
@@ -789,7 +789,7 @@ public void parseAdminResidentCommand(String[] split) throws TownyException {
 			Resident resident = townyUniverse.getDataSource().getResident(split[0]);
 
 			if (split.length == 1) {
-				TownyMessaging.sendMessage(getSender(), TownyFormatter.getStatus(resident, player));
+				TownyMessaging.sendMsg(getSender().getServer().getPlayer(getSender().getName()), TownyFormatter.getStatus(resident, player));
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:792]: This is a spot that sendMessage should be used. Return to how it was.

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 2]
```diff
@@ -904,7 +904,7 @@ public void parseAdminTownCommand(String[] split) throws TownyException {
 			
 			
 			if (split.length == 1) {
-				TownyMessaging.sendMessage(getSender(), TownyFormatter.getStatus(town));
+				TownyMessaging.sendMsg(getSender().getServer().getPlayer(getSender().getName()), TownyFormatter.getStatus(town));
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:907]: This is a spot that sendMessage should be used. Return to how it was.

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 3]
```diff
@@ -1221,7 +1226,7 @@ public void parseAdminNationCommand(String[] split) throws TownyException {
 			
 			Nation nation = townyUniverse.getDataSource().getNation(split[0]);
 			if (split.length == 1) {
-				TownyMessaging.sendMessage(getSender(), TownyFormatter.getStatus(nation));
+				TownyMessaging.sendMsg(getSender().getServer().getPlayer(getSender().getName()), TownyFormatter.getStatus(nation));
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:1229]: This is a spot that sendMessage should be used. Return to how it was.

**Review by LlmDl — CHANGES_REQUESTED** [review]
Some things which need to be changed, overall I think it will improve the look of the feedback messages where your changes are good.

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 4]
```diff
@@ -1221,7 +1226,7 @@ public void parseAdminNationCommand(String[] split) throws TownyException {
 			
 			Nation nation = townyUniverse.getDataSource().getNation(split[0]);
 			if (split.length == 1) {
-				TownyMessaging.sendMessage(getSender(), TownyFormatter.getStatus(nation));
+				TownyMessaging.sendMessage(getSender().getServer().getPlayer(getSender().getName()), TownyFormatter.getStatus(nation));
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:1229]: Return the full line to how it was before.

**Review by LlmDl — CHANGES_REQUESTED** [review]
There's still some major whitespacing issues. Some of the stuff needed to be changed in TownyAdminCommand were not completely done.

You definitely ought to rebase your branch off of master, so your changes are based off of the latest master.

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 5]
```diff
@@ -921,20 +921,25 @@ public void parseAdminTownCommand(String[] split) throws TownyException {
...
 				} catch (NotRegisteredException e) {
-					TownyMessaging.sendMessage(sender, Translation.of("msg_error_no_player_with_that_name", split[2]));
+					TownyMessaging.sendMsg(sender, Translation.of("msg_error_no_player_with_that_name", split[2]));
 					return;
 				}
+				
+				// If player is outlawed in target town, remove them from outlaw list.
+				if (town.hasOutlaw(resident))
+					town.removeOutlaw(resident);
+
```
[the remark this answers is not in the document]
Comment (GNosii) [reply src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:931]: rebased

**Pull request closed without merging**